stable/mlrun/values.yaml

```diff
@@ -69,6 +69,24 @@ api:
   # runAsNonRoot: true
   # runAsUser: 1000
 
+  readinessProbe:
+    httpGet:
+      path: /api/healthz
+      port: http
+    initialDelaySeconds: 10
+    timeoutSeconds: 30
+    periodSeconds: 5
+    failureThreshold: 10
```

Comment:
The timing combination here allows for a scenario where requests hang and pile up forever until something blows up
periodSeconds should be larger then timeout *  failure threshold

Reply:
I can see why period seconds should be larger than timeout seconds, but why multiply by failure threshold ?

Reply:
Changed timout to 10 and period to 15

Reply:
No.
It sends a request every `periodSeconds` no matter whether requests are failing or succeeding.
Each request timeouts after `timeoutSeconds`
It counts failures, and when there were more than `failureThreshold` failures in a row, it failes the probe.
You can see here that it's a simple ticker that fires every `periodSeconds` -https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/prober/worker.go#L150
Looking on the code I didn't see any retry logic like you're describing
you can see here that a timeout is counted as a successful probing - but a failure - https://github.com/kubernetes/kubernetes/blob/master/pkg/probe/http/http.go#L93